### DISCUSSION TITLE
[Feature] Display the number of photos for each person and place

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8680] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On the “Explore”, “People” and “Places” pages I would like to see a counter that would show an exact number of photos of each person and place.

Similar, but not the same issue - #5799

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5926
